transcom/mymove
# B20285-INT-tio review weights crash

## Agility ticket
[B20285](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20285)

## Summary
The change was to fix a UI crash when the "Review weight" button on the "Payment Request" tab as a TIO.  In addition, on the "Review Weights" page, the "File Not Found" was replaced with "No documents provided".  

### How to test
* Customer: Create a move with an PPM shipment
* SC: Approve move.
* TIO:
  - Search for move and select it.
  - Select "Payment Requests" tab
  - Click "Review weights" button
  - Confirm:
    *  the web page doesn't crash w/an "oops" screen
    * "No documents provided" in the document viewer.

## ANDI Screenshots
"No documents provided" is just text and not an actionable UI element.

## Testing Screenshots
none
